components/map/basic/src/leaflet/map.js

```diff
@@ -33,7 +33,10 @@ export default class LeafletMap {
   }
 
   dispatchFirstLoad () {
-    this.mapDOM.dispatchEvent(new window.CustomEvent('leaflet_map_loaded', { detail: this.getParamsForRequest() }))
+    this.dispatchCustomEvent({
```

Comment:
This method alreadu checks if can use window.CustomEvent or initCustomEvent and so on